content/doc/tutorials/build-a-node-js-and-react-app-with-npm.adoc

```diff
@@ -86,48 +71,40 @@ GitHub account and then cloning this fork locally.
    `git clone \https://github.com/YOUR-GITHUB-ACCOUNT-NAME/simple-node-js-react-npm-app` +
    where `YOUR-GITHUB-ACCOUNT-NAME` is the name of your GitHub account.
 
+=== Start your Jenkins instance
+
+1. Obtain the latest Jenkins instance, customized for this tutorial, by cloning the link:https://github.com/jenkins-docs/quickstart-tutorials.git[quickstart-tutorials repository].
+2. After cloning, navigate to the `quickstart-tutorials` directory, and execute the command `docker compose up -d node` to run the example.
+3. Once the containers are running successfully (you can verify this with `docker compose ps`), the controller can be accessed at http://localhost:8080.
+
+If you are unable to install `docker compose` on your machine for any reason, you can still run the example in the cloud for free thanks to link:https://www.gitpod.io/[GitPod]. GitPod link:https://www.gitpod.io/pricing[is free] for 50 hours per month.
+You need to link it to your GitHub account so you can run the example in the cloud.
+Click on link:https://gitpod.io/?autostart=true#https://github.com/jenkins-docs/quickstart-tutorials[that link] to open a new browser tab with a GitPod workspace where you'll be able to start the Jenkins instance, and run the rest of the tutorial.
```

Comment:
```suggestion
Utilize link:https://gitpod.io/?autostart=true#https://github.com/jenkins-docs/quickstart-tutorials[this link] to open a new browser tab with a GitPod workspace where you'll be able to start the Jenkins instance and run the rest of the tutorial.
```
Update link text, suggestion to use "utilize" instead of "click".